### ISSUE TITLE
tc.js uses different interfaces for up and down

### ISSUE THREAD

**Issue by ibc** [sitespeedio/throttle#22]
* For  `down` is calls `tc` with "ifb0": https://github.com/sitespeedio/throttle/blob/master/lib/tc.js#L61
* For `upd` it calls `tc` with `iFace`: https://github.com/sitespeedio/throttle/blob/master/lib/tc.js#L76

It's a bug, right?

**Comment by soulgalore** [comment]
Yeah, this  looks strange, and I guess it will not work if you are not using ifb0 then? I think this was ported directly from the  settings from WPT. Let me have look before I release the next version.
